### ISSUE TITLE
Query - Project, Team and Version support

### ISSUE THREAD

**Issue by raghurakesh** [Apicurio/apicurio-studio#786]
Hi @EricWittmann ,

First of all, I would like to thank you for creating this awesome tool. 

This is more of a query - In the roadmap of APICurio, I noticed that you will be supporting the Project, Team concepts and will also support Multiple versions of each project/API. Just wanted to check if there is an approximate timeline when you are planning to release this.

Thanks & Regards
Rakesh

**Comment by EricWittmann** [comment]
Glad you like it!  Please don't hesitate to suggest improvements and report bugs. :)

I don't have a specific timeline for those features, but I can tell you that they are high priority.  I would expect to see them within the next 2 to 3 months.  Especially the Project/Team concepts.  Collaboration is currently a great feature of Apicurio (users can simultaneously edit the same API design) but is hampered quite a bit by the limitations of how we currently do sharing.

**Comment by raghurakesh** [comment]
Hi @EricWittmann, 

A humble follow up. Would like to check if this is still in the roadmap? Is there any date planned for this? Please advise.

Thanks & Regards

**Comment by EricWittmann** [comment]
Yes this is still on the roadmap!  We've been sidetracked here for a few months with another project, which has delayed Apicurio Studio progress a bit.  Hopefully we'll be able to get back to making larger improvements soon!

**Comment by askfor** [comment]
Any new updates on this issue?

**Comment by mecseid** [comment]
Any update when we expect this feature? 
(or there is any issue that block this?)

**Comment by EricWittmann** [comment]
There's nothing blocking this other than available developer bandwidth.  :(

**Comment by v-bilous** [comment]
Hi @EricWittmann 

thanks for the tool, it's excellent, I'm looking at how to apply this to my codegen project. 
I miss the Project concept that gives the ability to combine Apis (rest, async) and add additional properties/settings.  

I think I will be implementing this (Project concept) in the Fork to have it available for my codegen. 
Do you have any writeups/ideas about the Project functionality in the Apicurio studio? I can take it into account so later may crate proper PR. 

Thanks,
Vova

**Comment by EricWittmann** [comment]
ping @Apicurio/developers - interesting info here.  I think this is consistent with our thinking around projects.  Although the last two bullets we haven't really talked about much.

Regarding bullet (2) - access management at the project level is very important.  I'm hoping we will have the bandwidth to also have Teams, since adding individual collaborators is slow and painful in the current Studio.

We really do need to think about how references will work - whether they are bound in some way to Projects or whether we allow references across Projects.  If we do **not** allow cross-project references, that might constrain the problem a bit.  But it makes it harder to handle the use case of organization-wide shared entities centrally defined.  That said, we could (and I think should) allow resources to exist in multiple Projects.  Food for thought anyway.

We're just starting to think about the new API now, so it's a good time to talk about this.

**Comment by v-bilous** [comment]
Hi Eric,

I've missed ur answer, sorry about that. I have an idea about references, let me review how it currently works and I will be able to add vision. 
I've paused work for some time, but for sure I wanna proceed with the instrument. 

Have a great week!
With the best wishes from Kyiv, Ukraine 🇺🇦 
Vova

**Comment by EricWittmann** [comment]
Thanks for the response @v-bilous - don't worry about delays, I **often** get way behind on my GitHub communications.
